rust/tw_aptos/src/signer.rs

```diff
@@ -0,0 +1,46 @@
+use std::marker::PhantomData;
+use std::str::FromStr;
+use tw_coin_entry::error::SigningResult;
+use tw_coin_entry::signing_output_error;
+use tw_keypair::ed25519;
+use tw_proto::Aptos::Proto;
+use crate::address::{Address, AptosAddress};
+use crate::transaction_builder;
+
+pub trait AptosContext {
+    type Address: AptosAddress;
+}
```

Comment:
I see that `EvmContext` confuses. The reason why we used `EvmContext` with an `EvmContext::Address` associated type is that there are some EVM forks that have different address formats.
For example, Ronin supports `ronin:ec49280228b0d05aa8e8b756503254e1ee7835ab`, but other EVM chains do not.
As far as I know, Aptos has only one address format (at least for now), so `AptosContext` and `AptosAddress` are not needed.
Therefore, the signer can be just:
```rust
pub struct Signer;

impl Signer {
  pub fn sign_proto_impl(...) {
    // ...
    let sender = Address::from_str(&input.sender)?;
    // ...
 }
}
```